external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -4361,7 +4337,7 @@ fn docstring_test() {
         "M6.move",
         "s: Symbols::M6::DocumentedStruct",
         Some((4, 11, "M6.move")),
-        Some("A documented function that unpacks a DocumentedStruct\n"),
+        None,
```

Comment:
This was a (harmless) bug actually - doc string should not be associated with a use of a local variable (as its definition cannot have one)